daemon/disk_usage.go

```diff
@@ -14,7 +14,7 @@ import (
 func (daemon *Daemon) ContainerDiskUsage(ctx context.Context) ([]*types.Container, error) {
 	ch := daemon.usage.DoChan("ContainerDiskUsage", func() (interface{}, error) {
 		// Retrieve container list
-		containers, err := daemon.Containers(&types.ContainerListOptions{
+		containers, err := daemon.Containers(context.TODO(), &types.ContainerListOptions{
 			Size: true,
 			All:  true,
 		})
```

Comment:
Thanks, look good. Do you mind addressing this in a follow up? The behaviour here doesn't change, and I would like to avoid blowing up this PR with a new package being vendored.

Reply:
SGTM! 👍